src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+// eslint-disable-next-line import/extensions
```

Comment:
- [ ] We strongly suggest not using `eslint disabler`. Could you remove it as well? 


> 🔧 Sometimes we may encounter challenges when using ESLint. Unexpected errors may arise which can be difficult to resolve quickly. If you come across an issue that is functioning properly but is still displaying an error message when uploaded to GitHub, I recommend disabling it in the .eslintrc.json file. This can help to prevent any confusion or delays in the development process. Let me know if you need any assistance.